src/mongocxx/collection.hpp

```diff
@@ -119,10 +118,8 @@ class MONGOCXX_API collection {
     ///
     /// @see http://docs.mongodb.org/manual/reference/command/aggregate/
     ///
-    cursor aggregate(
-        const pipeline& pipeline,
-        const options::aggregate& options = options::aggregate()
-    );
+    cursor aggregate(const pipeline& pipeline,
```

Comment:
I think we want to enforce the clang format. Would it be worth doing a full pass sometime?